### ISSUE TITLE
Observation / suggestion

### ISSUE THREAD

**Issue by MostHated** [RashadAnsari/gorilla#1]
Hey there,  
  Not to detract anything from your repo/work, but is gorilla the best name? I mean, there are already pretty famous packages out there using gorilla. (github.com/gorilla). Just seems like over utilizing an already well established name is just going to cause confusion and possibly frustration. Again, not disrespecting you or your work, it's just my thoughts on it. 

Thanks,
-MH

**Comment by xesina** [comment]
I agree please consider changing your repo name to prevent further confusions on the golang community :))
